lib/danger/danger_suggester.rb

```diff
@@ -0,0 +1,29 @@
+require 'danger'
```

Comment:
```suggestion
dummy
```




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>